src/test/java/com/microsoft/sqlserver/jdbc/CustomTrustManager/CustomTrustManagerTest.java

```diff
@@ -0,0 +1,62 @@
+package com.microsoft.sqlserver.jdbc.CustomTrustManager;
+
+import java.sql.DriverManager;
+
+import static org.junit.Assert.*;
+import org.junit.jupiter.api.Test;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+
+import com.microsoft.sqlserver.jdbc.SQLServerConnection;
+import com.microsoft.sqlserver.jdbc.SQLServerException;
+import com.microsoft.sqlserver.testframework.AbstractTest;
+
+@RunWith(JUnitPlatform.class)
+public class CustomTrustManagerTest extends AbstractTest {
+
+    /**
+     * Connect with a permissive Trust Manager that always accepts the X509Certificate chain offered to it. The test assumes PermissiveTrustManager
+     * class exists in the current package.
+     * 
+     * @throws Exception
+     */
+    @Test
+    public void testWithPermissiveX509TrustManager() throws Exception {
+        String url = connectionString + ";trustManagerClass=" + this.getClass().getPackage().getName() + ".PermissiveTrustManager" + ";encrypt=true;";
```

Comment:
There's no need for generating the class name from the package plus a string. You can directly reference the class and call getName(), i.e. TrustManagerWithConstructorArg.class.getName(). That way it'll be strongly typed and catch name changes to the dummy trust manager classes.